### PR TITLE
Databroker Configuration tests [12785]

### PR DESCRIPTION
**Pull request by jparisu**
Merge after #25

### CODE REVIEW

**Review by rsanchez15 — APPROVED** [review]

File: include/databroker/types/topic/WildcardTopic.hpp [hunk 1]
```diff
@@ -34,6 +34,9 @@ class WildcardTopic : public AbstractTopic
     //! Using parent constructos
     using AbstractTopic::AbstractTopic;
 
+    //! Constructor that allows any type
+    WildcardTopic(const std::string& topic_name);
```
Comment (rsanchez15) [inline include/databroker/types/topic/WildcardTopic.hpp:38]: Missing test for this function.

Comment (rsanchez15) [reply include/databroker/types/topic/WildcardTopic.hpp:38]: Just add some new cases to the topic tests as that PR is already merged.

File: test/unittest/configuration/DatabrokerConfigurationTest.cpp [hunk 2]
```diff
@@ -0,0 +1,602 @@
...
+    {
+        topic[TOPIC_NAME_TAG] = topic_name;
+    }
+
+    if (topic_type != "")
+    {
+        topic[TOPIC_TYPE_NAME_TAG] = topic_type;
+    }
+
+    yaml[list_tag].push_back(topic);
```
Comment (rsanchez15) [inline test/unittest/configuration/DatabrokerConfigurationTest.cpp:54]: The topic will be added even if there are no name or type given.

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

File: include/databroker/types/topic/WildcardTopic.hpp (hunk 1 above)
Comment (jparisu) [reply include/databroker/types/topic/WildcardTopic.hpp:38]: :ok:

File: test/unittest/configuration/DatabrokerConfigurationTest.cpp (hunk 2 above)
Comment (jparisu) [reply test/unittest/configuration/DatabrokerConfigurationTest.cpp:54]: This could actually happen, addeded to the tests

**Review by rsanchez15 — APPROVED** [review]

**Pull request merged**